File: week-04/day-03/sharpie.py
class Sharpie(object):
    def __init__(self, color, width):
        self.color = color
        self.width = width
        self.ink_amount = 100

    def use(self):
        self.ink_amount -= 10

sharpie_blue = Sharpie("blue", 5)

print(sharpie_blue.color, sharpie_blue.width, sharpie_blue.ink_amount)

sharpie_blue.use()

print(sharpie_blue.ink_amount)

File: week-08/day-01/sharpie.js
'use strict';

function Sharpie (color, width) {
    this.sharpieColor = color;
    this.sharpieWidth = width;
    this.inkAmount = 100;
    this.use = function() {
        while (this.inkAmount > 0) {
            this.inkAmount -= this.sharpieWidth;
        }
        if (this.inkAmount < 0){
            this.inkAmount = 0;
        }
    }
}

var sharpieOne =  new Sharpie('blue', 30);

console.log(sharpieOne.sharpieColor);
console.log(sharpieOne.inkAmount);

sharpieOne.use();

console.log(sharpieOne.inkAmount);